### PR TITLE
FYLE-86ctxu8p0-retrieve-virtual-cards

### PR DESCRIPTION
**Pull request by Kulsekar4**

### CODE REVIEW

File: src/components/schemas/virtual_card.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Specifies the valid till date of the card.
+      example: "2022-02-01"
+    creator_user_id:
+      type: string
+      description: |
+        The unique id of an user who created the card.
+      nullable: true
```
Comment (Kulsekar4) [inline src/components/schemas/virtual_card.yaml:68]: this has to be false

File: src/components/schemas/virtual_card.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Specifies the valid from date of the card.
+      example: "2022-02-01"
+    valid_till_at:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Specifies the valid till date of the card.
```
Comment (Kulsekar4) [inline src/components/schemas/virtual_card.yaml:62]: valid till date of the current balance of the card

File: src/components/schemas/virtual_card.yaml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      type: string
+      description: |
+        The unique id of the amex account to which the card is assigned
+      nullable: true
+      example: amactr7NhV9Qn6
+    valid_from_at:
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Specifies the valid from date of the card.
```
Comment (Kulsekar4) [inline src/components/schemas/virtual_card.yaml:56]: specifies from what date the card would be active

**Review by sumanth-fyle — APPROVED** [review]

File: src/components/schemas/virtual_card.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      type: string
+      description: |
+        Specifies the last 5 digits of the card.
+      example: '4543'
+    state:
+      type: string
+      description: |
+        Status of the virtual card
+      nullable: true
+      enum:
```
Comment (aditya999123) [inline src/components/schemas/virtual_card.yaml:30]: are these already agreed on ?

Comment (Kulsekar4) [reply src/components/schemas/virtual_card.yaml:30]: Yes @aditya999123 sir, got agreed [here](https://fylein.slack.com/archives/C05DTKNDC6Q/p1703053879762049?thread_ts=1702901104.005969&cid=C05DTKNDC6Q)

**Review by aditya999123 — APPROVED** [review]

**Pull request merged**